db/oracle_schema.rb

```diff
@@ -504,14 +504,6 @@
   add_index "deleted_objects", ["object_type", "object_id"], name: "index_deleted_objects_on_object_type_and_object_id"
   add_index "deleted_objects", ["owner_type", "owner_id"], name: "index_deleted_objects_on_owner_type_and_owner_id"
 
-  create_table "end_user_plans", force: :cascade do |t|
```

Comment:
How did you remove this from schema?

Reply:
I just deleted the line 😳

Reply:
> # This file is auto-generated from the current state of the database. Instead
> # of editing this file, please use the migrations feature of Active Record to
> # incrementally modify your database, and then regenerate this schema definition.

Reply:
OK I created a migration here https://github.com/3scale/porta/pull/1667